### PR TITLE
Add AR configuration and permissions dependency

### PR DESCRIPTION
**Pull request by HedwigAR**
This project is just great to immediately get started without the setup. But just like the person in issue #2 I also was a bit confused that the AR setup was not present in this project. Here's that added for both android and iOS.

I didn't want to add too much to the App.tsx so currently when you start XR mode you'll be in the middle of the cubes.

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -5,22 +5,65 @@
...
  */
 
-import React, { useState, FunctionComponent, useEffect } from "react";
-import { SafeAreaView, StatusBar, View, ViewProps } from "react-native";
-import { EngineView, useEngine } from "@babylonjs/react-native";
-import { SceneLoader } from "@babylonjs/core/Loading/sceneLoader";
-import { Camera } from "@babylonjs/core/Cameras/camera";
-import { ArcRotateCamera } from "@babylonjs/core/Cameras/arcRotateCamera";
-import "@babylonjs/loaders/glTF";
+import React, {
```
Comment (bghgary) [inline App.tsx]: I guess you are using some formatter (prettier maybe)? The formatting is all different. Maybe not a good idea to change it for this kind of PR.

**Review by bghgary — APPROVED** [review]
Other than the formatting, LGTM.

File: App.tsx (hunk 1 above)
Comment (HedwigAR) [reply App.tsx]: You're right, I missed that, I reverted the formatting changes.

**Comment by bghgary** [comment]
Hmm, I don't have write permissions for this repo and thus can't merge. @SergioRZMasson When you are back, can you check this? I'm assuming you have admin since we transferred this from your account.

**Pull request merged**